src/renderer/components/ft-input/ft-input.js

```diff
@@ -23,6 +23,10 @@ export default defineComponent({
       type: String,
       default: null
     },
+    maxlength: {
+      type: Number,
+      default: 100
```

Comment:
100 is just a number I chose that's arbitrarily high but minimizes UI breaking / misleading the user as to how much of their input will realistically be displayed when applicable. If anything, it's probably a bit too high, but I think it's fine. I don't think there's any reason to have different individualized values for each input.

Reply:
Places with 100 char limit: (13)
- src/renderer/views/UserPlaylists/UserPlaylists.vue - searchBar
- src/renderer/views/SubscribedChannels/SubscribedChannels.vue - searchBar
- src/renderer/views/History/History.vue - searchBar
- src/renderer/views/Channel/Channel.vue - searchBar
- src/renderer/components/proxy-settings/proxy-settings.vue - proxyPort
- src/renderer/components/playlist-info/playlist-info.vue - Playlist Name
- src/renderer/components/playlist-info/playlist-info.vue - Playlist Description
- src/renderer/components/playlist-info/playlist-info.vue - Search for Videos
- src/renderer/components/ft-profile-edit/ft-profile-edit.vue - Profile Name
- src/renderer/components/ft-playlist-add-video-prompt/ft-playlist-add-video-prompt.vue - searchBar
- src/renderer/components/ft-create-playlist-prompt/ft-create-playlist-prompt.vue - Playlist Name
- src/renderer/components/distraction-settings/distraction-settings.vue - Hide Channels
- src/renderer/components/distraction-settings/distraction-settings.vue - Hide Videos and Playlists Containing Text

Places with unlimited length: 12 (`:maxlength="null"`)


Youtube playlist name now has 150 char limit, description has 5000 char limit:
![image](https://github.com/FreeTubeApp/FreeTube/assets/1018543/4a16a6e6-ed78-4951-87dd-6035c4a2fb0f)

![image](https://github.com/FreeTubeApp/FreeTube/assets/1018543/9da8f71d-9e2b-42a0-92a8-e97a1e301873)


Channel name now has 100 char limit:
https://www.reddit.com/r/JackSucksAtLife/comments/17e8h08/since_when_was_the_youtube_channel_name_limit/
![image](https://github.com/FreeTubeApp/FreeTube/assets/1018543/fab73544-ac1b-4df3-8f19-427962e630f4)


This means these places should have 255 chars limit due to Playlist Name + Channel Name = 250 + 5 char for whatever purpose like delimiter:
- src/renderer/views/UserPlaylists/UserPlaylists.vue - searchBar
- src/renderer/components/playlist-info/playlist-info.vue - Playlist Name
- src/renderer/components/playlist-info/playlist-info.vue - Playlist Description (not saying it should match Youtube's 5000 char limit but the limit should be at least the same as playlist name or larger)
- src/renderer/components/ft-playlist-add-video-prompt/ft-playlist-add-video-prompt.vue - searchBar
- src/renderer/components/ft-create-playlist-prompt/ft-create-playlist-prompt.vue - Playlist Name

Other places:
- src/renderer/components/distraction-settings/distraction-settings.vue - Hide Videos and Playlists Containing Text (150 due to playlist anme)
- src/renderer/components/proxy-settings/proxy-settings.vue - proxyPort (max value 65535 = 5 chars)


And now there are 6 places left with 100 char limit while 12 places with no limit and I have no idea why there is a default value of 100

Reply:
I'm tired and having a bit of trouble following. What's the rationale for most of the suggested changes beyond conforming to YT? I did mean to remove the limit on the playlist description, but I must have accidentally removed it, so I can do that. My general idea is that we can use one arbitrary not-too-large value for most custom name-like fields and unbounded for ones where we don't have a reason to care.

Reply:
Re-reading this, I come off as unnecessarily callous. My main point is that I'm willing to spend at most ten minutes on further changes for this PR, because I do think we (and developers in the FOSS community as a whole) have a tendency to bikeshed about obscure minutia in a way that can be quite counterproductive. I worry that if I were to implement your suggested changes in full, I'm going to get a review from another person saying to do it all another way, and the process just gives me the heebie jeebies sometimes. So if you are confident that you have a set of changes that you think I can do pretty quickly that are also unlikely to be opposed by any other developer on our team, I'm game. Just give me the sign.

Reply:
Just for changes to be made: change char limit to 255 in following places:
- src/renderer/views/UserPlaylists/UserPlaylists.vue - searchBar
- src/renderer/components/playlist-info/playlist-info.vue - Playlist Name
- src/renderer/components/ft-playlist-add-video-prompt/ft-playlist-add-video-prompt.vue - searchBar
- src/renderer/components/ft-create-playlist-prompt/ft-create-playlist-prompt.vue - Playlist Name

Reasons: 
- Remote playlist can have 150 chars
- Channel name can have 100 chars (and some people might want to add them)
- Existing display logic truncate string to 255 chars